packages/expo-dev-launcher/bundle/hooks/useOnUpdatePress.tsx

```diff
@@ -1,19 +1,20 @@
 import * as React from 'react';
 
 import { Toasts } from '../components/Toasts';
+import { formatUpdateUrl } from '../functions/formatUpdateUrl';
 import { loadUpdate } from '../native-modules/DevLauncherInternal';
 import { useToastStack } from '../providers/ToastStackProvider';
 import { useUpdatesConfig } from '../providers/UpdatesConfigProvider';
 import { Update } from '../queries/useUpdatesForBranch';
 
 export function useOnUpdatePress() {
   const toastStack = useToastStack();
-  const { runtimeVersion, updatesUrl } = useUpdatesConfig();
+  const { runtimeVersion, projectUrl } = useUpdatesConfig();
```

Comment:
this was a bug that went unnoticed since the iOS module uses `Expo.plist` when this value is missing